src/dev/impl/DevToys/ViewModels/Tools/Converters/Timestamp/TimestampToolViewModel.cs

```diff
@@ -28,275 +43,193 @@ internal bool IsInputInvalid
             set => SetProperty(ref _isInputInvalid, value);
         }
 
-        internal double Timestamp
+        internal string DSTInfoText
```

Comment:
For future maintainers (and for me 😅), it would be nice to add a description at the top of every properties in this document. You can do it with the following syntax:

```
        /// <summary>
        /// Gets or sets the daylight saving time local time displayed in the UI (or whatever describes better this property 😉)
        /// </summary>
        internal string DSTInfoLocalDateTime
        {
            get => _dstInfoLocalDateTime;
            set => SetProperty(ref _dstInfoLocalDateTime, value);
        }
```